Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -177,6 +180,8 @@ static Vector<StyledMarkedText> coalesceAdjacentWithSameRanges(Vector<StyledMark
                 // If higher or same priority and opaque, override background color.
                 if (it->style.backgroundColor.isOpaque())
                     previousStyledMarkedText.style.backgroundColor = it->style.backgroundColor;
+                if (it->style.textShadow)
+                    previousStyledMarkedText.style.textShadow = WTFMove(it->style.textShadow);
```

Comment:
is this safe to move? (ie, is it->style.textShadow ever used later at some point)

Reply:
This function coalesces styles so after we set `previousStyledMarkedText` with the correct styles, `it` should never be used again and we will iterate to the next `StyledMarkedText`.